src/coreclr/jit/codegencommon.cpp

```diff
@@ -1785,6 +1785,10 @@ void CodeGen::genGenerateMachineCode()
         {
             printf("; Tier-1 compilation\n");
         }
+        else if (compiler->IsTargetAbi(CORINFO_NATIVEAOT_ABI))
```

Comment:
I did it in https://github.com/dotnet/runtime/pull/75425 just completely forgot about that PR 😄

Reply:
Ah, I can revert it from here now that you've remembered :-)